include/SkipProbe/SkipProbe.hpp

```diff
@@ -13,6 +11,11 @@ namespace SkipProbe
 
 #include "Iterator.hpp"
 
+// Visual Studio 2019 requires explicitly including <stdexcept> to see the definition of std::out_of_range.
+#if defined(_MSC_VER) && _MSC_VER >= 1920
+	#include <stdexcept>
```

Comment:
Is there any harm including this header unconditionally?

Reply:
Apparently not.  PR updated.